panel/io/document.py

```diff
@@ -16,6 +16,7 @@
 from bokeh.application.application import SessionContext
 from bokeh.document.document import Document
 from bokeh.document.events import DocumentChangedEvent, ModelChangedEvent
+from bokeh_django.consumers import WSConsumer
```

Comment:
I'm hoping there is a better condition to test in the change made below, but if using `WSConsumer` ends up being the best case then maybe we want to conditionally import it here so `panel` doesn't have to depend on the `bokeh_django` repo.